components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementAdminService.java

```diff
@@ -169,10 +204,16 @@ public IdentityProvider getIdentityProvider(String federatedIdPName)
      * @return local authenticators array
      * @throws org.wso2.carbon.identity.application.common.IdentityApplicationManagementException
      */
-    public LocalAuthenticatorConfig[] getAllLocalAuthenticators()
-            throws IdentityApplicationManagementException {
-        applicationMgtService = ApplicationManagementService.getInstance();
-        return applicationMgtService.getAllLocalAuthenticators(getTenantDomain());
+    public LocalAuthenticatorConfig[] getAllLocalAuthenticators() throws IdentityApplicationManagementException {
+
+        try {
+            applicationMgtService = ApplicationManagementService.getInstance();
+            return applicationMgtService.getAllLocalAuthenticators(getTenantDomain());
```

Comment:
since we know the tenantDomain using getTenantDomain() method. Shall we include that in the error log?